administrator/components/com_associations/views/associations/tmpl/default.php

```diff
@@ -0,0 +1,166 @@
+		. $aComponent . '&aview=' . $aView . '&id=';
+	}
+}
+?>
+
+<?php if ($assoc) : ?>
+
+	<form action="<?php echo JRoute::_('index.php?option=com_associations&view=associations'); ?>" method="post" name="adminForm" id="adminForm">
+		<div id="j-main-container">
+
+		<?php echo JLayoutHelper::render('joomla.searchtools.default', array('view' => $this, 'options' => array('filterButton' => false))); ?>
+
+		<?php if (empty($this->items)) : ?>
+	        <div class="alert alert-no-items">
+	            <?php echo JText::_('JGLOBAL_NO_MATCHING_RESULTS'); ?>
+	        </div>
+	    <?php else : ?>
+			<table class="table table-striped" id="contactList">
+				<thead>
+					<tr>
+						<th class="nowrap">
+							<?php echo JHtml::_('searchtools.sort', 'JGLOBAL_TITLE', 'title', $listDirn, $listOrder); ?>
+						</th>
+						<th width="15%" class="nowrap hidden-phone">
+							<?php echo JHtml::_('searchtools.sort', 'JGRID_HEADING_LANGUAGE', 'language_title', $listDirn, $listOrder); ?>
+						</th>
+						<?php if ($assoc) : ?>
+						<th width="5%" class="nowrap hidden-phone hidden-tablet">
+							<?php echo JHtml::_('searchtools.sort', 'COM_ASSOCIATIONS_HEADING_ASSOCIATION', 'association', $listDirn, $listOrder); ?>
+						</th>
+						<?php endif;?>
+						<th width="1%" class="nowrap hidden-phone">
+							<?php echo JHtml::_('searchtools.sort', 'JGRID_HEADING_ID', 'a.id', $listDirn, $listOrder); ?>
+						</th>
+					</tr>
+				</thead>
+				<tfoot>
+					<tr>
+						<td colspan="10">
+							<?php //echo $this->pagination->getListFooter(); ?>
+						</td>
+					</tr>
+				</tfoot>
+				<tbody>
+				<?php
+				/**
+					@todo ACL Check
+				*/
+				$n = count($this->items);
+				foreach ($this->items as $i => $item) :
+					$canCreate  = $user->authorise('core.create', $componentFilter);
+					$canEdit    = $user->authorise('core.edit', $componentFilter);
+					$canCheckin = $user->authorise('core.manage', 'com_checkin') || $item->checked_out == $user->get('id')|| $item->checked_out == 0;
+					$canChange  = $user->authorise('core.edit.state', $componentFilter) && $canCheckin;
+					?>
+					<tr class="row<?php echo $i % 2; ?>">
+						<td class="nowrap has-context">
+							<div class="pull-left">
+								<?php if ($canEdit || $canEditOwn) : ?>
+									<a href="<?php echo JRoute::_($link . (int) $item->id); ?>"><?php echo $this->escape($item->title); ?></a>
+								<?php else : ?>
+									<?php echo $this->escape($item->title); ?>
+								<?php endif; ?>
+							</div>
+						</td>
+						<td class="small hidden-phone">
+							<?php echo $item->language_title ? JHtml::_('image', 'mod_languages/' . $item->language_image . '.gif', $item->language_title, array('title' => $item->language_title), true) . '&nbsp;' . $this->escape($item->language_title) : JText::_('JUNDEFINED'); ?>
+						</td>
+						<?php if ($assoc) : ?>
+						<td class="hidden-phone hidden-tablet">
```

Comment:
this is the main field of this view IMHO shoulnd't not be hidden in mobile/tablet view hidden-phone hidden-tablet